src/composables/useActiveSession.js

```diff
@@ -74,6 +80,16 @@ export function useActiveSession() {
 			console.info('Session has been marked as active')
 		} catch (error) {
 			console.error(error)
+			if (error?.response?.status === 404) {
+				// In case of 404 - participant did not have a session, block UI to join call
+				store.dispatch('updateLastJoinedConversationToken', '')
+				if (experimentalRecoverSession) {
+					// Automatically try to join the conversation again
+					store.dispatch('joinConversation', { token: token.value })
+				} else {
+					showError(t('spreed', 'Failed to join the conversation.') + '\n' + messagePleaseTryToReload)
```

Comment:
this is not joining, it's making as active

Reply:
dropping user toast for now